### PR TITLE
Feature/splash page

### PR DESCRIPTION
**Pull request by maarcingebala**
This PR adds the splash page.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: templates/home/index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        <div class="content-container">
+          <div class="splash__container">
+            {% include "home/images/splash.svg" %}
+          </div>
+          <main class="main">
+            <h1>Congratulations.</h1>
+            <h2>Your Saleor instance is ready to work</h2>
+
+            <p>
+              Now you can start setting up your ecommerce. Below are some useful
```
Comment (NyanKiyoshi) [inline templates/home/index.html:170]:
```suggestion
              Now you can start setting up your e-commerce. Below are some useful
```

File: templates/home/index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <nav class="nav">
+        {% include "home/images/logo.svg" %}
+        <span> <a href="#">View documentation</a> for Saleor 2.10 </span>
```
Comment (NyanKiyoshi) [inline templates/home/index.html:158]: Hard-coded version is fine?

Comment (NyanKiyoshi) [inline templates/home/index.html:158]: URL is `#`

Comment (maarcingebala) [inline templates/home/index.html:158]:
```suggestion
        <span> <a href="https://docs.saleor.io/docs/index/" target="_blank">View documentation</a> for Saleor 2.10 </span>
```

Comment (maarcingebala) [reply templates/home/index.html]: It's "good enough".

**Pull request merged**